test/fancylogger.py

```diff
@@ -532,9 +532,14 @@ def test_zzz_fancylogger_as_rootlogger_logging(self):
         msg = 'this is my string'
         logging.debug(msg)
 
-        # fails on python 3.11
-        self.assertEqual(stringfile.getvalue(), '',
-                         msg="logging.debug reports nothing when fancylogger loglevel is debug")
+        if sys.version_info < (3,7):
+            # logging stream handling was changed in python 3.7
+            # this is a good thing as messages can no longer go missing
+            self.assertEqual(stringfile.getvalue(), '',
```

Comment:
factor out the common parts. the expected value is the only difference?

Reply:
that, and the message, I'll fix it